app-script-payslip-sender/PayslipSender.js

```diff
@@ -0,0 +1,26 @@
+function payslipSender() {
+  var spreadsheet = SpreadsheetApp.getActive().getSheetByName('List');
+  var lastRow = spreadsheet.getLastRow();
+  var dataRange = spreadsheet.getRange("A2:D" + lastRow).getValues();
+  Logger.log(dataRange);
+
+  for(var i=0; i < dataRange.length; i++) {
+    var employeeData = dataRange[i];
+    var employeeName = employeeData[1];
+    var salary = employeeData[2];
+    var email = employeeData[3];
+
```

Comment:
put a check if all the fields have non-empty values. No need to do anything in case these fields are empty.